docs/Home.md

```diff
@@ -2,41 +2,77 @@
+
+Some have since then created their own repositories external to Quarks, but others just never happeend because not everyone wants the overhead of maintaining a build setup while in other cases we actually had an idea for an extension but not solidified enough yet to be included in core.
+
+Quarkiverse is our proposed solution to create a "home" for such extensions/projects.
+
+#What is Quarkiverse
+
+This Quarkiverse github organization provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+
+Quarkus extensions hosted in the Quarkiverse organization will by default be included into the Quarkus extensions catalog displayed on [code.quarkus.io](http://code.quarkus.io) and the Quarkus command line tools (such as `mvn quarkus:list-extensions`, `gradle listExtensions`). To stay listed only requirement
+is that the extension keeps functioniong, stays uptodate and cause no harm.
 
 # Why Quarkiverse
 
 In the early days and actually up until recently, Quarkus extensions contributed by the community members (including the core Quarkus team) have been welcome in the Quarkus core repository at https://github.com/quarkusio/quarkus. Today the core repository has grown a lot and it's evident that it simply becomes troublesome to work with it for its contributors. That and other maintenance and infrastructure related issues inspired the creation of a separate organization called Quarkiverse to host Quarkus extension projects contributed by the community (including the Quarkus core team members themselves).
 
+The advantages to join Quarkiverse are:
+
+  . Automated and secured publishing of your maven releases to Maven Central.
+  . Automated Cross-testing of your extension with Quarkus builds/releases (see https://github.com/quarkusio/quarkus-ecosystem-ci)
+  . Inclusion in registry used by Quarkus tooling to browse extensions (i.e. code.quarkus, command line and IDE tools).
+  . Consistent formatting and release of project
+  . Quarkus team members can help and fix issues if project has build issues.
+  
+Note: While the Quarkus project has the "keys" to publish under `io.quarkiverse`, each project in Quarkiverse is driven and maintained by 
+the lead of that project. They decide what happens with the project - Quarkiverse will just expect releases to be compatible with Quarkus - if a project
+for some reason cannot do a release we can in the best scenario help or in the worst scenario we'll temporarily exclude the project from the registry to avoid
+end-users to have a bad experience.
+ 
+If you do not want to commit to the above guidelnes then you are more than welcome to publish an extension outside of Quarkus or Quarkiverse; and then you can
+still be listed in the registry - it is just not handled automtically. In this case we do please request you use your own group id to clearly identify those artifacts are from a third-party.
```

Comment:
"In this case we do please request you use..." I'd just say "In this case please use..." It'll be easier to read.